.ruby-version

```diff
@@ -1 +1 @@
-2.3.3
\ No newline at end of file
+2.5.1
```

Comment:
Can we also update the [dockerfile](https://github.com/ministryofjustice/et-admin/blob/develop/Dockerfile#L1) to use the `2.5.1-webapp-onbuild` image?

Reply:
oh yeah !!